rules/qc.smk

```diff
@@ -253,14 +253,24 @@ def get_alignment_qc(sample):
     output.append(f"{{qc_dir}}/dedup/{{{{assembly}}}}-{sample}.samtools-coordinate.metrics.txt")
     output.append(f"{{qc_dir}}/dedup/{{{{assembly}}}}-{sample}.samtools-coordinate.samtools_stats.txt")
 
-    # add insert size metrics
-    if config['layout'][sample] == "PAIRED":
-        output.append(f"{{qc_dir}}/InsertSizeMetrics/{{{{assembly}}}}-{sample}.tsv")
+    if "atac_seq" in get_workflow():
```

Comment:
why not for the other workflows anymore? MT DNA ratio seems a useful QC mark for all of them to me, or is it to cumbersome?

Reply:
I haven't seen seen anyone mention this for RNA or CHiP. For ATAC it is important since it cuts open DNA. And mitochondrial DNA is very open, and there are many mitochondria. You want to make sure you didnt only pcr your mtochondrial reads. We could keep it, but I prefer the multiqc reports to contain pipeline specific outputs.

Reply:
Let's just keep it for now, but ideally the multiqc reflects the pipeline that was run.

Reply:
sure. this was a question, not a request :)